test/uk/gov/hmrc/traderservices/journey/TraderServicesFrontendFormatSpec.scala

```diff
@@ -201,6 +201,17 @@ class TraderServicesFrontendFormatSpec extends UnitSpec {
           )
         )
       )
+      validateJsonFormat(
+        """{"state":"AnswerImportQuestionsMandatoryVesselInfo","properties":{"declarationDetails":{"epu":"123","entryNumber":"100000Z","entryDate":"2020-10-05"},"importQuestionsAnswers":{"requestType":"New","priorityGoods":"LiveAnimals","freightType":"RORO"}}}""",
+        State.AnswerImportQuestionsMandatoryVesselInfo(
+          DeclarationDetails(EPU(123), EntryNumber("100000Z"), LocalDate.parse("2020-10-05")),
+          ImportQuestions(
+            requestType = Some(ImportRequestType.New),
```

Comment:
Will need to be requestType.hold (the test underneath will also need to be updated from anything other than hold)

Reply:
fixed